src/app/views/jobs/jobs.component.ts

```diff
@@ -43,10 +44,12 @@ export class JobsComponent extends TranslationListener implements OnInit {
   }
 
   loadData() {
+    this.loadingJobs = true;
     this.jobProxy.getJobs({ include: 'owner,company,hourly_pay,company.company_images', 'filter[filled]': false, 'page[number]': this.page.toString() })
       .then(result => {
         this.jobs = result.data;
         this.totalJobs = result.total;
+        this.loadingJobs = false;
```

Comment:
👍 for readability! nice that this barely required any changes to the jobs component 👑